src/main/java/de/phoenixmitx/lombokextensions/ArrayExtension.java

```diff
@@ -149,6 +149,10 @@ public <T> T[] add(T[] arr, int index, T element) {
 		return newArr;
 	}
 
+	public <T> T[] add(T[] arr, T... c) {
+		return addAll(arr, c);
+	}
+	
```

Comment:
Do not add another method that just calls another method.
Consider adding a custom implementation or adding the spread operation to the addAll method.
The change should also be made to all primitive types listed in this file.